Add SPECIES_IPC to command RIGs

Lorwp opened the pull request:
🆑 lorwp
tweak: IPCs can now use the Command Hardsuits
/🆑

So human-shaped IPCs can use Human-shaped RIGs.

~~Assumed to be intended due to the cooling unit installed in the suits. Doesn't give them anymore access mechanically to use anything other than the Command RIG unless they're a traitor~~

Fixes #22353

![metal boi wearing that command rig](https://user-images.githubusercontent.com/8052896/50436130-62af8780-0938-11e9-9b90-bf4e277d43c4.png)

Sbotkin commented:
>Assumed to be intended due to the cooling unit installed in the suits.

Not really, it was mostly for FBPs and for all of the suits, not just command.

Sbotkin commented:
But since we have IPCs as Bridge Officers, it was a good idea, wait! Also Eckles has nothing to do with it, it's not a lore thing.

Lorwp commented:
Branch is still there, will look at this again. Trying to figure out the runtimes this is getting

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
On Thursday, December 27, 2018 12:09 AM, Sbotkin <notifications@github.com> wrote:

> But since we have IPCs as Bridge Officers, it was a good idea, wait! Also Eckles has nothing to do with it, it's not a lore thing.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, [view it on GitHub](https://github.com/Baystation12/Baystation12/pull/24086#issuecomment-449964681), or [mute the thread](https://github.com/notifications/unsubscribe-auth/AHrgoO6_W5UDb36JtWa-EAaTnrJgNoEJks5u83UdgaJpZM4ZhcaE).

Lorwp commented:
Runtimes are unrelated, this is good to go after review 👍

Lorwp commented:
Rebased onto 2067fc629cb3ae4dc04ba42dec22258e9fec36d0

also, bump.

Commit:
Add SPECIES_IPC to command RIGs 

So human-shaped IPCs can use Human-shaped RIGS.

Actually not intended but with BOs being IPCs this is important.

Now with auto-changelog

afterthought2 reviewed (approved):
I don't know who this even affects. If anyone thinks this is bad, speak now.

Sbotkin commented:
It affects beep-boop Bridge Officers and only them.

Lorwp commented:
Yeah, just Bridge Officers. Though if for some reason they're given the same access as any Line Officer they can use the other HCMs as well
